clayui.com/src/utils/getSection.js

```diff
@@ -20,10 +20,6 @@ const sortByOrderAndTitle = (a, b) => {
 	}
 };
 
-const navigationRelationships = {
-	'/docs/components/form.html': [],
-};
```

Comment:
this is now no longer needed since we can just check if any of the child elements have the path as a parent